src/solidClientHelpers/utils.js

```diff
@@ -143,7 +143,7 @@ export function chain(object, ...operations) {
 }
 
 export function defineThing(options, ...operations) {
-  return chain(createThing({ options }), ...operations);
+  return chain(createThing(options), ...operations);
```

Comment:
Please extend the test of createThing to verify that the options are passed correctly.